metricbeat/module/elasticsearch/pending_tasks/_meta/fields.yml

```diff
@@ -0,0 +1,25 @@
+- name: pending_tasks
+  type: group
+  description: >
+    pending_tasks
+  fields:
+    - name: max_time_in_queue_millis
+      type: long
+      description: >
+        Max time in queue (ms)
+    - name: pending_tasks_priority_high
+      type: long
+      description: >
+        Number of pending tasks in priority high
+    - name: pending_tasks_priority_urgent
```

Comment:
In general check naming conventions here: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html